idb/postgres/postgres.go

```diff
@@ -2549,7 +2598,8 @@ func (db *IndexerDb) yieldApplicationsThread(ctx context.Context, rows *sql.Rows
 		var paramsjson []byte
 		var created *uint64
 		var closed *uint64
-		err := rows.Scan(&index, &creator, &paramsjson, &created, &closed)
+		var deleted *bool
```

Comment:
why not sql.NullBool here? (and a couple other places; usage inconsistent)

Reply:
I'm not sure actually, this is the pattern I used for CreatedAt/DeletedAt so I copied it here.